python/nx-cugraph/README.md

```diff
@@ -2,31 +2,78 @@
 
 ## Description
 [RAPIDS](https://rapids.ai) nx-cugraph is a [backend to NetworkX](https://networkx.org/documentation/stable/reference/classes/index.html#backends)
-with minimal dependencies (`networkx`, `cupy`, and `pylibcugraph`) to run graph algorithms on the GPU.
+to run supported algorithms with GPU acceleration.
 
-### Contribute
+## System Requirements
 
-Follow instructions for [contributing to cugraph](https://github.com/rapidsai/cugraph/blob/branch-23.12
-and [building from source](https://docs.rapids.ai/api/cugraph/stable/installation/source_build/), then build nx-cugraph in develop (i.e., editable) mode:
-```
-$ ./build.sh nx-cugraph --pydevelop
-```
+Using nx-cugraph with this notebook requires the following:
```

Comment:
references the notebook. Not sure if this is a cut/paste thing or if you meant to reference the notebook in github which I think would be great.

Reply:
Yep, copy/paste error, thanks.  I'm going to push an update soon.